aiida_common_workflows/cli/options.py

```diff
@@ -48,6 +49,21 @@ def get_spin_types():
     return [entry.value for entry in SpinType]
 
 
+class JsonParamType(click.ParamType):
```

Comment:
```suggestion
class JsonParamType(click.StringParamType):
```
this will then automatically make sure that the value passed is a string